tools/codegen/gen_backend_stubs.py

```diff
@@ -25,6 +25,7 @@ def parse_backend_yaml(
         grouped_native_functions: Sequence[Union[NativeFunction, NativeFunctionsGroup]],
         backend_indices: Dict[DispatchKey, BackendIndex]
 ) -> ParsedExternalYaml:
+    _GLOBAL_PARSE_NATIVE_YAML_CACHE.clear()
```

Comment:
One consequence of https://github.com/pytorch/pytorch/pull/68530/files#r754702093 is that I can't modify this cache directly in `test_gen_backend_stubs.py` - I think it shows up as a separate variable when I use the `importlib` machinery. It seems... okay to clear the cache here, because at least right now, you'll never want to be in a situation where you want to call `parse_backend_yaml()` multiple times on the same cached yaml output.